index.js

```diff
@@ -19,5 +19,24 @@ module.exports = function(source) {
         this.cacheable();
     }
 
+    // if force sourcemap requested embed inline sources (helps resolve issues with
+    // transpiled languages like typescript
+    if (userOptions['force-sourcemap']) {
```

Comment:
Yea that's the one, though with webpack I'm a bit fuzzy with what point the source maps are output